media/css/pebbles/components/_global-nav.scss

```diff
@@ -515,6 +531,7 @@ html.moz-nav-open .moz-global-nav-page-mask {
 
         .download-title {
             font-weight: bold;
+            white-space: nowrap;
```

Comment:
Well, for really long strings this could create overlapping text at some screen sizes, right? If that's a possibility, I'd say wrapping may be a bit more robust?